android/app/src/main/java/app/organicmaps/MwmActivity.java

```diff
@@ -1239,7 +1239,7 @@ public void onPlacePageDeactivated()
   @SuppressWarnings("unused")
   public void onSwitchFullScreenMode()
   {
-    if ((mPanelAnimator && mPanelAnimator.isVisible()) ||UiUtils.isVisible(mSearchController.getToolbar()))
+    if ((mPanelAnimator && mPanelAnimator.isVisible()) || UiUtils.isVisible(mSearchController.getToolbar()))
```

Comment:
🤦 Sorry, it's Java, not C++. Let's return the comparison back, polish commits, and merge.
```
/home/runner/work/organicmaps/organicmaps/android/app/src/main/java/app/organicmaps/MwmActivity.java:1242: error: bad operand types for binary operator '&&'
    if ((mPanelAnimator && mPanelAnimator.isVisible()) || UiUtils.isVisible(mSearchController.getToolbar()))
                        ^
  first type:  PanelAnimator
  second type: boolean
```